src/elm/Page/Community/ActionEditor.elm

```diff
@@ -559,37 +560,17 @@ update msg model ({ shared } as loggedIn) =
         NoOp ->
             UR.init model
 
-        CompletedLoadCommunity community ->
+        CompletedLoadObjectives community objectives ->
             if community.creator == loggedIn.accountName then
-                -- Check the action belongs to the objective
                 let
                     maybeObjective =
-                        List.filterMap
-                            (\o ->
-                                if o.id == model.objectiveId then
-                                    Just o
-
-                                else
-                                    Nothing
-                            )
-                            community.objectives
-                            |> List.head
+                        List.find (.id >> (==) model.objectiveId) objectives
```

Comment:
took a minute to got what was going there `>>` operator is weird

Reply:
in a good way 👽

Reply:
Yeah, point-free can be confusing sometimes.

In this case I kind of like how it reads if you just ignore all the punctuation:
`find [something where] id == model.objectiveId`